packages/sdk/contractkit/src/wrappers/Reserve.ts

```diff
@@ -71,7 +71,7 @@ export class ReserveWrapper extends BaseWrapper<Reserve> {
   getAssetAllocationSymbols = proxyCall(
     this.contract.methods.getAssetAllocationSymbols,
     undefined,
-    (symbols) => symbols.map(this.kit.web3.utils.hexToAscii)
+    (symbols) => symbols.map(this.connection.web3.utils.hexToAscii)
```

Comment:
again, maybe extract this to the connection package, to be able to easily remove it in the future